dataland-frontend/tests/e2e/utils/FakeFixtureUtils.ts

```diff
@@ -151,6 +151,14 @@ export class Generator {
     return faker.company.buzzNoun();
   }
 
+  randomParagraph(): string | null {
+    return this.valueOrNull(this.guaranteedParagraph());
+  }
+
+  guaranteedParagraph(): string {
```

Comment:
This is the first time I see this guaranteed notation. We would usually call this `generateParagraph`.

Reply:
Also, docs missing. But sonar will tell you as well.

Reply:
Docs are not enforced here and other functions also don't have docs here.

Reply:
The other functions (which also exist in main) all use "guaranteed" here.   Do you mean to change them all?

Reply:
Don't mind the first comment.